Introduce Alignment Meetings

zupo opened the pull request:
During the discussion on https://github.com/niteoweb/operations/issues/457
we realized that our Retrospetives are too long and some Niteans are getting
bored during them. Especially during User Story Demos, which take a long time
to play and are from all parts of Niteo. Marketers need to watch development
demos and vice-versa.

We propose instead to make the Retrospective shorter and only about the
scrum process. Then when we are done with the retrospective, only those who
are interested gather again to watch demos and review spreadsheets and charts.
The rest of Niteans can get back to their work.

karantan commented on work-process/scrum.md:
```diff
@@ -58,11 +58,23 @@ This is also the time to bring up any other comments about the work process in g
+The Scrum Master will create a new [retrospective issue](https://github.com/niteoweb/operations/issues/new?template=retrospective.md&title=Retrospective%20for%20Sprint%20#) before each retrospective. This issue has a template that serves as a checklist for the meeting and helps the Scrum Master write up a brief report afterwards.
+
+
+## Orientation Meeting
```
`Dharma Initiative: Orientation Meeting` 😄 

P.S. Sorry I could not help myself ^^ 😄

zupo replied:
😄

zupo commented:
As per discussions during https://github.com/niteoweb/operations/issues/478 I updated this PR with the following changes:
* Renamed Orientation to Alignment meeting. Orientation usually means the initial onboarding for newcomers.
* We keep playing demos during the Retrospective as they build context for later discussion about the sprint goods/bads/improvements.
